python/sglang/srt/managers/scheduler.py

```diff
@@ -346,12 +354,17 @@ def event_loop_normal(self):
                         self.update_running_batch()
                         if not self.running_batch:
                             break
+                        if self.server_args.enable_dp_mla:
+                            self.check_dp_batch_running(batch)
                         result = self.run_batch(batch)
                         self.process_batch_result(batch, result)
             else:
                 self.check_memory()
                 self.new_token_ratio = self.init_new_token_ratio
 
+            if batch and batch.forward_mode.is_idle():
```

Comment:
why is this needed?

Reply:
Avoid to change the last batch here. But the last batch seems not used in the normal loop. We can remove this check.